store/src/main/java/com/dropbox/android/external/store4/DiskWrite.kt

```diff
@@ -2,9 +2,9 @@ package com.dropbox.android.external.store4
 
 interface DiskWrite<Raw, Key> {
     /**
-     * @param key to use to get data from persister
-     * If data is not available implementer needs to
-     * either return Observable.empty or throw an exception
+     * @param key to use to get data from persister.
```

Comment:
This interface has nothing to do with a presister. I don't think it makes sense to mention it in the doc.
from the impl we have it should throw or return false if the write failed

Reply:
actually we never check the return value. it should just throw an exception when failing it seems

Reply:
done